### PR TITLE
Feat: BECS-13 Login

### PR DESCRIPTION
**Pull request by nhidh**

### CODE REVIEW

**Comment by quangtran85** [comment]
@nhidh 
These commit's comments should start by `fix: ` prefix.
```
feat: BECS-13 fix logic expired and fix eslint
feat: BECS-13 fix bug
```

File: src/apps/auth/handlers/authorizer.ts [hunk 1]
```diff
@@ -75,7 +74,7 @@ const loginCheck = async (userId: string, token: string) => {
     accessToken: token,
     userId: userId,
     expired: {
-      $lte: moment().toDate(),
+      $gte: moment().toDate(),
```
Comment (quangtran85) [inline src/apps/auth/handlers/authorizer.ts:77]: why do you use `$gte` here?

File: src/apps/auth/services/auth.service.ts [hunk 2]
```diff
@@ -63,15 +66,40 @@ export class AuthService {
...
+  async refresh(data: RefreshInput): Promise<ResourceDataOutput<TokenOutput>> {
+    const verified: any = verifyJwtToken(data.refreshToken);
+    if (!verified?.userId) {
+      const { errorCode, message, httpCode } = Errors.INVALID_REFRESH_TOKEN;
+      throw new AppException(errorCode, message, httpCode);
+    }
+
+    const token = await this.tokenRepository.findOne({
+      refreshToken: data.refreshToken,
+    });
```
Comment (quangtran85) [inline src/apps/auth/services/auth.service.ts:88]: If the refresh token has been removed once the user logged out, what should we do at here? :)

Comment (nhidh) [reply src/apps/auth/services/auth.service.ts:88]: I fixed it [8986ffa](https://github.com/quangtran85/serverless-tutorial/pull/8/commits/8986ffaf721bb378f2f89d91bd210f95d8c5d35b)

File: src/apps/auth/handlers/authorizer.ts (hunk 1 above)
Comment (nhidh) [reply src/apps/auth/handlers/authorizer.ts:77]: The code logic is wrong here
https://github.com/quangtran85/serverless-tutorial/pull/8/files#diff-b1580df8d02abbda65771a90a931cd8ca0e67e444f14746fe2fdff4567d0cd56R150
As a result, getting data here would no longer be correct, so I fixed it

**Review by quangtran85 — APPROVED** [review]
LGTM!

**Pull request merged**